src/main/java/com/tortu/api/configuration/JaxRSConfig.java

```diff
@@ -13,6 +13,7 @@ public JaxRSConfig() {
         log.info("Initializing Rest Services...");
         packages("com.tortu.api");
         register(UsuarioRestService.class);
+        register(RecetaRestService.class);
```

Comment:
👍  Ideally, if the line 14 is there, it will scan all classes, and you can remove everything from line 16, but you can keep both :)